### DISCUSSION TITLE
Local version works great, deployed version doesn't work

### DISCUSSION THREAD

**Discussion by NickHodges** [Pagefind/pagefind#609] [Q&A] [answered]
Hey, if I run my site locally using `serve`, search works great. 

But when I deploy to Vercel -- everything builds and deploys fine. But the search doesn't work. 

You can see it in action here:  https://nickhodges.com  selecting the magnifying glass in the upper right.

What might be the reason?

**Comment by bglw** [comment]
Hmm, can you share any build logs from your Vercel build? Looks like the `/pagefind/` directory files are 404ing on the live site, so Pagefind might not be running successfully during the Vercel build.

**Reply by NickHodges** [reply, marked as answer]
Ahh!  That was helpful!

So I looked closely at the build logs, and I saw this:

`09:57:13 [build] directory: /vercel/path0/.vercel/output/static/`

so I set my `postbuild` to this:  

`"postbuild": "pagefind --site .vercel/output/static/"`

and now it works! 

Thank you for your help.  I'm grateful to you.  All the way from New Zealand!  Kia Ora!

**Reply by bglw** [reply]
Tēnā koe! Glad to hear it's all working! 😊
